packages/near-fast-auth-signer-e2e-tests/firebase.ts

```diff
@@ -0,0 +1,74 @@
+import admin from 'firebase-admin';
+
+import { serviceAccount } from './utils/serviceAccount';
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount as admin.ServiceAccount)
+});
```

Comment:
I'm not sure it is good idea to leave it at top level. Say if one of dev is working on a test that doesn't need firebase, this part of code doesn't need to be executed. I would rather initialise at before test